### ISSUE TITLE
Undefined method parse

### ISSUE THREAD

**Issue by unansweredocd** [kylekatarnls/business-day#103]
Thanks for creating this. 

I use the https://github.com/bmewburn/vscode-intelephense language server and whenever I pull in this package in my laravel project none of the original carbon methods are recognized anymore.

The most prominent example is calling Carbon::parse(), which will give me a undefined method parse warning.

Here's how to reproduce.

```
composer create-project laravel/laravel example-app
cd example-app
composer require cmixin/business-day

```

Then simply edit the App\Models\User.php file, create a new method such as:

```php
public function item()
{
    Carbon::parse();
}
```

If you need more info, please let me know.

**Comment by Tarpsvo** [comment]
I have the same issue with the exact same setup. Seems like the _ide_ types classes provided completely override the Carbon's own definitions. :(

**Comment by kylekatarnls** [comment]
For the record, PHPStorm merge both. It sounds to be a matter of how the IDE interprets it.

**Comment by unansweredocd** [comment]
Yeah, seems like I don't have the issue if I switch to phpactor's language server. I'll open an issue on the intelephense repository to see what that guy says. I'll get back to this issue once I have a response from him.

**Comment by unansweredocd** [comment]
I'm going to go ahead and close this as it seems to be a language server problem.
